### DISCUSSION TITLE
Unable to browse source members in the files

### DISCUSSION THREAD

**Discussion by sessterling** [codefori/vscode-ibmi#434] [Q&A]
Hello, 

First of all, many thanks to the team for this fantastic tool that you provide. 
I just discovered and installed it this afternoon.

However I struggle with the following problem : I can only display the content of some source files (very few). When I unfold the others, nothing happens (no source members displayed, no error message).
I thought that there is something wrong with the files, so I copied some source member to a file that used to work,  and then same problem on this file : no member displayed anymore !

So the problem seems to come from the members... But I have no idea what it could be.

I will be grateful for any suggestion.

Kind regards
Pascal

**Comment by worksofliam** [comment]
Hi @sessterling

1. do you have SQL enabled or disabled? You can check that in the connection settings.
2. How many members are in the source file you're trying to look at?

**Reply by pabloto** [reply]
Hi, 

I've the same problem, I discovered then the problem was the in my source i put some hexadecimal value, then produce this effect in seu:

![image](https://user-images.githubusercontent.com/9719313/145463117-8003dbff-e508-49e5-a010-d0d736d16bc9.png)

e.g. X'22'  put the line in highlight mode

but to use it in visual studio code I have to remove that.

**Reply by worksofliam** [reply]
@sessterling Are you having an issue opening the source member list or a specific source member?

**Comment by sessterling** [comment]
Hi @worksofliam and @pabloto 

Thanks for your quick reply  :-) 
In the meantime I guess I found the problem thanks the error messages that are now displayed (since I restarted VSC once again) : each time I try to unfold a file that doesn't show anything, I get a message like "Invalid closing quote : got "s" at line xxxx instead of delimiter ...."

I made some tests in a small file (11 members) : in one member's text I had double-quotes (").  I removed them and it works. 

So i seems that if there is an unaccepted symbol in one member text, it won't display anything,  which seems normal as it certainly "breaks" the browse command.
Perhaps it is possible to escape symbols that make problems  (quote, double quote, ...)   before calling the browse command ?

**Reply by worksofliam** [reply]
@sessterling You never answered about whether or not you had SQL enabled - is it?

**Reply by sessterling** [reply]
Sorry @worksofliam ,
Yes SQL is enabled.

**Reply by worksofliam** [reply]
@sessterling Great!

What version of db2util do you have installed on the remote server?

You can check with `yum list installed`
